server/api/translation/openAI.js

```diff
@@ -1,46 +1,46 @@
-import path from "path";
 import dotenv from "dotenv";
 import { Configuration, OpenAIApi } from "openai";
 import { fileURLToPath } from "url";
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
-dotenv.config({ path: __dirname + "/../../.env" });
+dotenv.config({ path: __dirname + "/../.env" });
 
-const sendPromptToOpenAi = async (openai, prompt) => {
-  return await openai.createChatCompletion({
-    model: "gpt-3.5-turbo",
-    messages: [{ role: "user", content: prompt }],
+const init = () => {
+  const config = new Configuration({
+    apiKey: process.env.OPEN_AI_API_KEY,
   });
+  return new OpenAIApi(config);
 };
 
-export const checkProfanity = async (msg) => {
```

Comment:
why did you delete this?

Reply:
it's ok , i changed it last week, and this issue is before 2 weeks